src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -351,6 +351,20 @@ public override string GetDetailString()
                 BaseHexSize);
     }
 
+    protected override Dictionary<Compound, float> CalculateTotalReproductionCost()
+    {
+        var result = new Dictionary<Compound, float>(BaseReproductionCost);
```

Comment:
One small thing I think I just noticed: shouldn't this call `base.CalculateTotalReproductionCost`? Because now this just skips the base reproduction cost...

So this line should be `var result = base.CalculateTotalReproductionCost()`.